install_config/syncing_groups_with_ldap.adoc

```diff
@@ -229,7 +229,7 @@ $ oadm groups prune --sync-config=config.yaml --confirm
 ----
 
 [[sync-examples]]
-== Sync Examples
+== Direct Membership Sync Examples
```

Comment:
Except that doing crazy group nesting is common with AD which is probably the most used LDAP server...